observability/logging/logging-6.0/log6x-configuring-lokistack-otlp.adoc

```diff
@@ -0,0 +1,150 @@
+
+* Resource
+* Scope
+* Log
+
+This allows metadata to be set for multiple entries simultaneously or individually as needed.
+
+include::modules/log6x-configuring-lokistack-otlp-data-ingestion.adoc[leveloffset=+1]
+
+== Attribute mapping
+
+Loki splits the configuration for mapping OTLP attribuites to stream labels and structured metadata into two places:
+
+*  `default_resource_attributes_as_index_labels` in the link:https://grafana.com/docs/loki/latest/configure/#distributor[`distributor` configuration]
+* `otlp_config` in the link:https://grafana.com/docs/loki/latest/configure/#limits_config[`limits_config`]
+
+By default, `default_resource_attributes_as_index_labels` maps certain resource attributes to stream labels within Loki. 
+
+Since this distributor configuration only applies to resource-level attributes and generates stream labels as output, the `otlp_config` setting allows mapping resource, scope, or log-level attributes to structured metadata.
+
+[NOTE]
+====
+Attributes that remain unmapped to either stream labels or structured metadata do not store in Loki.
+====
+
+The Loki Operator handles attribute configuration differently than Loki itself. Notably, the Loki Operator does not distinguish between `distributor` and `limits` configurations. Instead, the Loki Operator consolidates all attributes in the `limits` configuration. For more infomation on Operator-specific configuration options, see xref:../../../observability/logging/logging-6.0/log6x-configuring-lokistack-otlp.adoc#custom-attribute-mapping[Custom attribute mapping].
+
+=== Loki Operator defaults
+
+When using the Loki Operator in `openshift-logging` mode, default attribute mappings follow OpenShift configurations. For more information on configuring custom attributes within `openshift-logging` mode, see xref:../../../observability/logging/logging-6.0/log6x-configuring-lokistack-otlp.adoc#customizing-openshift-defaults[Customizing OpenShift defaults].
+
+=== Custom attribute mapping
+
+In `openshift-logging` mode, custom attribute mappings can be configured globally for all tenants or for individual tenants as needed. When custom mappings are defined, OpenShift defaults are bypassed. If default labels are still required, they must be included in the custom configuration. For more information, see xref:../../../observability/logging/logging-6.0/log6x-configuring-lokistack-otlp.adoc#customizing-openshift-defaults[Customizing OpenShift defaults].
+
+[NOTE]
+====
+A major difference between the Loki Operator and Loki itself lies in inheritance handling. Loki only copies `default_resource_attributes_as_index_labels` to tenants by default, while the Loki Operator applies the entire global configuration to each tenant in `openshift-logging` mode.
+====
+
+Within `LokiStack`, attribute mapping configuration is managed through the `limits` setting:
+
+[source,yaml]
+----
+# [...]
+spec:
+  limits:
+    global:
+      otlp: {} # <1> 
+    tenants:
+      openshift-logging:
```

Comment:
Although we say tenant **mode** is `openshift-logging`, the three tenants are `application`, `infrastructure` and `audit`. I suggest to use `application` here for the example code.